catalog/plugin/catalog-plugin-query-monitor-commands/src/main/java/ddf.catalog.querymonitorcommands/QueryMonitorCommandPrintTable.java

```diff
@@ -0,0 +1,187 @@
+     * Prints out a table representing a {@link QueryMonitorPlugin}'s ActiveSearchTable including information
+     * on all {@link ActiveSearch}'s currently populating the table
+     *
+     * @return always returns {@code null}
+     */
+
+    @Override
+    public Object execute() throws Exception {
+        try {
+            printSearchTableToConsole();
+        } catch (Exception e) {
+            LOGGER.debug(
+                    "Exception encountered in doExecute of QueryMonitorCommandPrintActiveSearcHTable.java!"
+                            + e.getMessage());
+        }
+        return null;
+    }
+
+    /**
+     * Helper method called by the execute method when the catalog:printtable command is called by the shell.
+     * Prints out a table representing a {@link QueryMonitorPlugin}'s ActiveSearchTable including information
+     * on all {@link ActiveSearch}'s currently populating the table
+     */
+
+    void printColor(Ansi.Color color, String message) {
+        String colorString;
+        if (color == null || color.equals(Ansi.Color.DEFAULT)) {
+            colorString = Ansi.ansi()
+                    .reset()
+                    .toString();
+        } else {
+            colorString = Ansi.ansi()
+                    .fg(color)
+                    .toString();
+        }
+        console.print(colorString);
+        console.print(message);
+        console.println(Ansi.ansi()
+                .reset()
+                .toString());
+    }
+
+    /**
+     * Called by the execute method when the catalog:printtable command is called by the shell.
+     * Prints out a table representing a {@link QueryMonitorPlugin}'s ActiveSearchTable including information
+     * on all {@link ActiveSearch}'s currently populating the table
+     */
+
+    void printSearchTableToConsole() {
+
+        if (queryMonitor1 == null) {
+            LOGGER.error(
+                    "Cannot print search table. QueryMonitorImpl not yet instantiated! Cannot printSearchTableToConsole()!");
+            return;
+        }
+        HashMap<UUID, ActiveSearch> activeSearchTable = queryMonitor1.getSearchTable();
+        if (activeSearchTable == null) {
+            LOGGER.error(
+                    "Cannot print search table. Retrieving ActiveSearchTable from queryMonitor returned null!");
+            return;
+        }
+
+        Iterator it = activeSearchTable.entrySet()
+                .iterator();
+        ActiveSearch as;
+
+        console.println();
+
+        String colorString = Ansi.ansi()
+                .fg(Ansi.Color.CYAN)
+                .toString();
+
+        console.print(colorString);
+        console.print(String.format(formatString,
+                "User Info",
+                "Source",
+                "Search Criteria",
+                "Start Time",
+                "Unique ID"));
+        console.println(Ansi.ansi()
+                .reset()
+                .toString());
+
+        while (it.hasNext()) {
```

Comment:
Also if You're not using the `key` part of the iterator, maybe just use `activeSearchTable.values()`? And if you collapse the formatted string logic into the as per below comment (https://github.com/codice/ddf/pull/554/files#r53536043) then you could do some fancy dancy stuff like foreach.
`activeSearchTable.values().forEach(search -> console.print(search.toFormattedString))`